run

```diff
@@ -89,6 +89,13 @@ EOF
   if [ "${RSYSLOG_LOG_TO_FILE}" == 'yes' ] ; then
     echo 'mail.* -/var/log/mail.log' >> /etc/rsyslog.conf
   fi
+
+  if [ ! ${RSYSLOG_REMOTE_PORT} ] ; then
+    RSYSLOG_REMOTE_PORT=514
+  fi
+  if [ ${RSYSLOG_REMOTE_IP} ] ; then
```

Comment:
Above we use `-z` to check for empty value, i guess this works also? but maybe nice to be consistent

Reply:
You're right.